xml/Microsoft.VisualBasic.Vsa/VsaEngine.xml

```diff
@@ -406,7 +407,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets a value that indicates whether the source is compiled.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the script engine is set to produce debug information when the <see cref="M:Microsoft.VisualBasic.Vsa.VsaEngine.Compile" /> method is called; otherwise, <see langword="false" />.</value>
```